packages/rule-engine/src/rules/__tests__/deployment-deny-rule.test.ts

```
@@ -1,17 +1,24 @@
 
+export type ResolvedRelease = {
+  id: string;
+  createdAt: Date;
+  version: {
+    id: string;
+    tag: string;
+    config: Record<string, any>;
+    metadata: Record<string, string>;
+  };
+  variables: Record<string, unknown>;
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)